eng/Directory.Packages.props

```diff
@@ -58,14 +58,13 @@
     <!--
       Visual Studio
     -->
-    <PackageVersion Include="Microsoft.VisualStudio.SDK" Version="18.0.1306-preview.1" />
+    <PackageVersion Include="Microsoft.VisualStudio.SDK" Version="18.0.2101-preview.1" />
     <PackageVersion Include="Microsoft.Internal.VisualStudio.Shell.Framework" Version="18.0.1211-preview.1" />
     <PackageVersion Include="Microsoft.ServiceHub.Client" Version="4.2.1017" />
     <PackageVersion Include="Microsoft.VisualStudio.Extensibility.Sdk" Version="18.0.52-preview1" />
     <PackageVersion Include="Microsoft.VisualStudio.Extensibility" Version="18.0.52-preview1" />
     <PackageVersion Include="Microsoft.VisualStudio.Extensibility.JsonGenerators.Sdk" Version="18.0.52-preview1" />
     <PackageVersion Include="Microsoft.VSSDK.Debugger.VSDConfigTool" Version="17.13.1100801-preview" />
-    <PackageVersion Include="Microsoft.VisualStudio.Imaging" Version="18.0.1306-preview.1" />
```

Comment:
These were duplicating entries from below, which means the script update doesn't work.